packages/steiger-plugin-fsd/src/forbidden-imports/README.md

```diff
@@ -172,3 +172,12 @@ flowchart BT
 ## Rationale
 
 This is one of the main rules of Feature-Sliced Design, it ensures low coupling and predictability in refactoring.
+
+## Related Rules
+
+If you need more granular control, this rule's functionality is split into two separate rules that you can use independently:
+
+- `no-higher-level-imports`: Only checks for imports from higher layers (e.g., features importing from pages)
+- `no-cross-imports`: Only checks for cross-imports between slices on the same layer (e.g., one entity importing from another entity)
+
+Using these separate rules allows you to enforce only specific aspects of the import restrictions or configure them differently.
```

Comment:
suggestion (non-blocking): I think it would be nice here to also include a little code snippet to instantly achieve the desired behaviour. Perhaps we can even put it instead of this last paragraph:

```js
// steiger.config.js
import { defineConfig } from 'steiger'
import fsd from '@feature-sliced/steiger-plugin'

export default defineConfig([
  ...fsd.configs.recommended,
  {
    rules: {
      'fsd/forbidden-imports': 'off',
      'fsd/no-cross-imports': 'error',
      'fsd/no-higher-level-imports': 'error',
    }
  },
  {
    // Allow cross-imports between widgets, for example
    files: ['src/widgets/**'],
    rules: {
      'fsd/no-cross-imports': 'off',
    }
  },
])
```